Kinvey-Xamarin/Store/DataStore.cs

```diff
@@ -101,6 +101,9 @@ public void SetCustomRequestProperty (string key, JObject value)
 			this.customRequestProperties.Add (key, value);
 		}
 
+		KinveyRealtimeDelegate<T> RealtimeCallback { get; set; }
+		KinveyRealtimeDelegate<string> routerCallback;
```

Comment:
You are correct, it does not.  Will remove this from both the `DataStore` and `Stream` classes